commit 4724b50f9e093cfa9bea7caa5f25f00755bcc568
Author: Manu Gautam <mgautam@codeaurora.org>
Date:   Tue Oct 16 12:52:07 2018 +0530

    phy: qcom-qusb2: Fix HSTX_TRIM tuning with fused value for SDM845
    
    [ Upstream commit c88520db18ba0b9a41326c3b8680e7c09eb4c381 ]
    
    Tune1 register on sdm845 is used to update HSTX_TRIM with fused
    setting. Enable same by specifying update_tune1_with_efuse flag
    for sdm845, otherwise driver ends up programming tune2 register.
    
    Fixes: ef17f6e212ca ("phy: qcom-qusb2: Add QUSB2 PHYs support for sdm845")
    Signed-off-by: Manu Gautam <mgautam@codeaurora.org>
    Reviewed-by: Douglas Anderson <dianders@chromium.org>
    Reviewed-by: Stephen Boyd <swboyd@chromium.org>
    Acked-by: Vivek Gautam <vivek.gautam@codeaurora.org>
    Signed-off-by: Kishon Vijay Abraham I <kishon@ti.com>
    Signed-off-by: Sasha Levin <sashal@kernel.org>

diff --git a/drivers/phy/qualcomm/phy-qcom-qusb2.c b/drivers/phy/qualcomm/phy-qcom-qusb2.c
index 9d6c88064158..69c92843eb3b 100644
--- a/drivers/phy/qualcomm/phy-qcom-qusb2.c
+++ b/drivers/phy/qualcomm/phy-qcom-qusb2.c
@@ -231,6 +231,7 @@ static const struct qusb2_phy_cfg sdm845_phy_cfg = {
 	.mask_core_ready = CORE_READY_STATUS,
 	.has_pll_override = true,
 	.autoresume_en	  = BIT(0),
+	.update_tune1_with_efuse = true,
 };
 
 static const char * const qusb2_phy_vreg_names[] = {
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1067_4724b50f9e093cfa9bea7caa5f25f00755bcc568.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1067_4724b50f9e093cfa9bea7caa5f25f00755bcc568.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
